src/lib/transactions.ts

```diff
@@ -9,10 +9,16 @@ import type { Token } from "~tokens/token";
 import BigNumber from "bignumber.js";
 import browser from "webextension-polyfill";
 import { balanceToFractioned, formatFiatBalance } from "~tokens/currency";
+import { AF_ERROR_QUERY } from "~notifications/utils";
+import { gql } from "~gateways/api";
+import { txHistoryGateways } from "~gateways/gateway";
+import { retryWithDelay } from "~utils/retry";
 
 let tokens: TokenInfo[] = null;
 export let tokenInfoMap = new Map<string, TokenInfo | Token>();
 
+const AGENT_TOKEN_ADDRESS = "JwjzLSbwIDIouY0iIMyVEbjMbVMOkL2TTEzdJWwtly8";
```

Comment:
I think the AGENT token address is `8rbAftv7RaPxFjFk5FGUVAVCSjGQB4JHDcb9P9wCVhQ`